sencha-workspace/packages/local/slate-assets/src/assets/controller/Assets.js

```diff
@@ -0,0 +1,1352 @@
+                ticketsaved: me.onAssetsTicketGridChange
+            }
+        });
+
+        me.getAssetsTemplateDataFieldsStore().load();
+
+    },
+
+    buildNavPanel: function() {
+        return this.getNavPanel();
+    },
+
+    // route handlers
+
+    showAllAssets: function() {
+        var me = this,
+            ExtHistory = Ext.util.History,
+            store = me.getAssetsStore(),
+            proxy = store.getProxy(),
+            manager = me.getManager();
+
+        ExtHistory.suspendState();
+        Ext.suspendLayouts();
+
+        // queue store to load
+        if (proxy.lastRequest) {
+            Ext.Ajax.abort(proxy.lastRequest)
+        }
+        // proxy.abortLastRequest(true);
+
+        delete proxy.extraParams.q;
+
+        //sync search form & treepanel
+        me.getSearchField().setValue(null);
+        me.syncNavTreePanel();
+
+        // activate manager
+        me.expandNavOnRender();
+        me.getApplication().getController('Viewport').loadCard(manager);
+
+        // resume layouts and insert a small delay to allow layouts to flush before triggering store load so loading mask can size correctly
+        Ext.resumeLayouts(true);
+
+        Ext.defer(function() {
+            //execute search forcing reload, since there are no "dirty" proxy params.
+            me.doSearch(true, function() {
+                ExtHistory.resumeState(false);
+            });
+        }, 10);
+    },
+
+    showAssets: function() {
+        var me = this,
+            treePanel = me.getNavTreePanel(),
+            navPanel = me.getNavPanel(),
+            searchField = me.getSearchField(),
+            rootAssetNode = me.getAssetsTreeNodesStore().getRootNode().getChildAt(0),
+            selectedNode = treePanel.getSelectionModel().getSelection()[0],
+            _selectRootNode = function() {
+                if (selectedNode && selectedNode.isAncestor(rootAssetNode))
+                    return;
+
+                treePanel.getSelectionModel().select(rootAssetNode, false, true);
+            };
+
+        Ext.suspendLayouts();
+
+        me.expandNavOnRender();
+        me.getApplication().getController('Viewport').loadCard(me.getManager());
+        Ext.resumeLayouts(true);
+
+        //reset search field & empty grid store
+        searchField.setValue(null);
+        me.getGrid().getStore().getProxy().setExtraParam("q", null);
+        me.getGrid().store.removeAll();
+
+        if (treePanel.rendered) {
+            _selectRootNode();
+        } else {
+            treePanel.on('render', _selectRootNode);
+        }
+    },
+
+    showAsset: function(assetId) {
+        var me = this,
+            ExtHistory = Ext.util.History,
+            manager = me.getManager(),
+            id = ExtHistory.decodeRouteComponent(assetId),
+
+            _onAssetNotFound = function() {
+                ExtHistory.resumeState(false);
+//                Ext.resumeLayouts(true);
+                debugger;
```

Comment:
@nbey do you recall why this `debugger` got left in here? Was there something you were still working on here we should leave a note about before removing this?